app/support/notification_sender.rb

```diff
@@ -17,24 +26,19 @@ def perform
 
       raise ActiveRecord::Rollback if @errors.any?
 
-      find_accounts_to_notify if SettingsHelper.has_review_period?
       mark_order_details_as_reviewed
       notify_accounts
     end
 
     @errors.none?
   end
 
-  def account_ids_notified
-    @account_ids_to_notify.map(&:first)
+  def accounts_notified_size
+    account_ids_to_notify.count
   end
 
   def accounts_notified
-    Account.find(account_ids_notified)
-  end
-
-  def accounts_notified_size
-    account_ids_to_notify.count
+    Account.where(id: account_ids_to_notify)
```

Comment:
I don't think we'll hit it, but should we protect from Oracle's 1000+ restriction?